src/bg_mpl_stylesheets/colors.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+
+class Colors(str, Enum):
+    bg_blue = "#0B3C5D"
+    bg_red = "#B82601"
+    bg_green = "#1C6B0A"
+    bg_light_blue = "#328CC1"
+    bg_light_grey = "#A8B6C1"
+    bg_yellow = "#D9B310"
+    bg_brown = "#6C5050"
+    bg_burgundy = "#76323F"
+    bg_olive_green = "#626E60"
+    bg_muted_olive = "#918770"
+    bg_beige = "#C09F80"
+    bg_grey = "#B0B0B0FF"
+    # Add more colors as needed
+
+    @classmethod
+    def get_color_name(cls, hex_value: str) -> str:
+        for color in cls:
+            if color.value.lower() == hex_value.lower():
+                return color.name
+        raise ValueError("Unknown color name. Please check Colors enum for available colors.")
```

Comment:
Would a non programmer user know how to do this? I think this is too jargony to be helpful.

Reply:
How about this?

```
raise ValueError("You have entered an unknown color hex value."
                         " Please check the src/bg_mpl_stylesheets/colors.py file for the available colors.")
```

You have entered an unknown color hex value -> reason for the error
Please check the src/bg_mpl_stylesheets/colors.py file for the available colors -> what to do fix it

Reply:
Yes, much better error message.  Even better would be a method that lists the available colors maybe?

Reply:
Ok!

Reply:
Wait, we already have that method, so could we mention that in the error message as an option? Look in the file or run that method?